admin/banner_manager.php

```diff
@@ -142,7 +142,7 @@
           'status' => $status,
           'banners_open_new_windows' => $banners_open_new_windows,
           'banners_on_ssl' => $banners_on_ssl,
-          'banners_sort_order' => (int)$banners_sort_order,
```

Comment:
I don't think you should remove this.

Reply:
Even though it's already an integer from line 107?
`$banners_sort_order = (int)$_POST['banners_sort_order'];`